projects/ngx-d3timeline/src/lib/options/selectors/type-options.selectors.ts

```diff
@@ -16,13 +13,19 @@ export const selectTypeActivityOptions = createSelector(
   typeOptions => getTypeActivityOptions.bind(null, typeOptions)
 );
 
-export const selectTypeActivityPadding = createSelector(
-  selectTypeActivityOptions,
-  typeActivityOptions => getTypeActivityPadding.bind(null, typeActivityOptions)
-);
+export const selectTypeActivityOption = <T>(
```

Comment:
should be able to type this similarly to the other ones

Reply:
done as part of #470